src/dist.rs

```diff
@@ -315,6 +345,79 @@ impl Dist {
     }
 }
 
+#[cfg(feature = "scss")]
+fn scss(
+    static_dir: &std::path::Path,
+    dist_dir: &std::path::Path,
+    options: sass_rs::Options
+) -> Result<()> {
+    use walkdir::{DirEntry, WalkDir};
+
+    fn is_scss(path: &std::path::Path) -> bool {
+        matches!(
+            path.extension().map(|x| x.to_str()).flatten(),
+            Some("sass") | Some("scss")
+        )
+    }
+
+    fn should_ignore(entry: &DirEntry) -> bool {
+        entry
+            .file_name()
+            .to_str()
+            .map(|x| x.starts_with("_"))
+            .unwrap_or(false) || entry.path().is_dir()
+    }
+
+    log::trace!("Generating dist artifacts");
+    let walker = WalkDir::new(&static_dir).into_iter();
+    for entry in walker
+        .filter_map(|x| match x {
+            Ok(x) => {
+                if !should_ignore(&x) {
```

Comment:
This is actually incorrect as it will ignore any file starting with `_` even if it's not a sass/scss file. This convention with the `_` is for sass/scss, it's not a unix convention or anything